market_nextdoor_api/models.py

```diff
@@ -68,10 +68,24 @@ class Preorder(models.Model):
     fulfilled = models.BooleanField(default=False)
     date_created = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
+    
+    
+#ManytoMany Test Models
+class Preorder_test(models.Model):
+    customer = models.ForeignKey(Customer, on_delete=models.CASCADE, null=False)
+    items = models.ManyToManyField(Item, through='Preorder_testItem')
```

Comment:
this is looking familiar with the use of "Through" in terms of has many items through preorder_items.